README.md

```diff
@@ -36,6 +36,16 @@ You can also roll back migrations by doing
 docker-compose run api alembic downgrade
 ```
 
+### Seeding data
+
+To populate development database with seed data, which is located in `/backend/database/seed_data` directory, run:
```

Comment:
aaah i don't wanna be a nag but... before doing this, people should check if the `seed.py` file has the right file configured (in this case, the `SYMPTOMS_FILE` variable), since it only processes one file. I'm leaving you an approved review because it's a totally dismissable comment that we can add further along the way.